pkg/distributor/forwarding/forwarding_test.go

```diff
@@ -118,6 +118,85 @@ func TestForwardingSamplesSuccessfullyToCorrectTarget(t *testing.T) {
 	))
 }
 
+func TestForwardingSamplesSuccessfullyToSingleTarget(t *testing.T) {
+	ctx := context.Background()
+	now := time.Now().UnixMilli()
+	forwarder, reg := newForwarder(t, testConfig, true)
+
+	url, reqs, bodiesFn, closeFn := newTestServer(t, 200, true)
+	defer closeFn()
+
+	rules := validation.ForwardingRules{
+		"metric1": validation.ForwardingRule{Ingest: false},
```

Comment:
How about we make one rule specify an invalid forwarding target, to check that it gets overriden by the one passed into `Forward()`?

```suggestion
		"metric1": validation.ForwardingRule{Ingest: false, Endpoint: "invalid"},
```